### PR TITLE
Enhancement of Navbar 

### PR DESCRIPTION
**Pull request by guntarakesh**
image
image
image

### CODE REVIEW

**Comment by Harshal141** [comment]
This is not the correct way of storing png.
Please read about how to organize react-app, where the assets are stored etc.

File: client/src/components/Home.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import MDEditor from "@uiw/react-md-editor";
 import rehypeSanitize from "rehype-sanitize";
-import Navbar from "./Navbar";
+import Navbar from "./navbar";
```
Comment (Harshal141) [inline client/src/components/Home.js:4]: Please change the file name to `Navbar`, and apply the changes wherever needed.

File: client/src/components/navbar.js [hunk 2]
```diff
@@ -1,66 +1,43 @@
...
+import dashboard1 from './icons8-menu-40.png'
 
 const Navbar = () => {
+
+  const [tr, setTr] = useState(false);
+
+  const handleClick = () => {
+    setTr(!tr);
+  }
+
```
Comment (Harshal141) [inline client/src/components/navbar.js:17]: Can you use good naming conventions.
Like i should be able to undestand the functions or var by its name

**Comment by guntarakesh** [comment]
@Harshal141 ok I will work on it .

**Comment by guntarakesh** [comment]
@Harshal141  i updated the needful changes please review it .

**Pull request merged**